src/Expose/main.tex

```diff
@@ -43,28 +41,65 @@
+	
+		\subsection{Inhalt der Umfrage}
+			Um ein möglichst unvoreingenommenes Ergebnis zu erhalten wird die Umfrage in vier Bereiche aufgeteilt:
+		   
+			\paragraph{Bewusste Nutzung}
+				Die Umfrage wird mit abstrakten Fragen zur Nutzung von Open Source Software beginnen. Zunächst soll ermittelt werden, ob der Nutzer/die Nutzerin weiß, was Open Source Software ist.
+			
+				Um im weiteren Verlauf der Umfrage sinnvolle Antworten zu erhalten, wird an dieser Stelle die Definition von Software und von Open Source Software erklärt.
+				% TODO: Gegebenenfalls werden hier noch weitere Definitionen nach einem Test hinzugefügt.
+			
+			
+			\paragraph{Unbewusste Nutzung}
+				Im zweiten Abschnitt werden konkrete Beispiele für Open Source Software genannt, um den Nutzer/die Nutzerin  darauf Aufmerksam zu machen, wo überall Open Source Software vorkommt, ohne dass es ihm/ihr aufgefallen ist. Dabei wird sowohl auf direkte als auch indirekter Nutzung eingegangen.
+			
+				Außerdem soll geklärt werden können, warum es Differenzen, wenn vorhanden, zwischen bewusster und unbewusster Nutzung gibt.
+			
+			\paragraph{Gründe für und gegen Open Source Software}
+				Im dritten Abschnitt werden Fragen bezüglich der Nutzungs- und Hinderungsgründe von Open Source Software gestellt. Dazu wird gefragt, warum oder warum nicht Endnutzer oder Unternehmen Open Source Software einsetzten. Die möglichen Antworten werden aus der Schweizer Studie zum Thema Open Source\footcite{oss:studie} genommen, um die Wahrnehmung mit der realen Nutzung zu vergleichen.
+			
+			\paragraph{Demographische Differenzen}
+				Um zu analysieren, ob es innerhalb der Bevölkerungen Differenzen gibt, soll die Umfrage eine Frage zur aktuellen Tätigkeit\footnote{Schüler, Student, Arbeitnehmer, etc.} und eine Frage zur Selbsteinschätzung der Computerkenntnisse enthalten.
+				
+		\subsection{Verbreitung}
+			Um die Reichweite der Umfrage zu maximieren wird diese in Form einer Online-Befragung umgesetzt. Dabei werden Plattformen wie Social Media, E-Mail, Messenger genutzt. Zudem wird der Link zu der Umfrage über private Kontakte in Schulen und an der Nordakademie verbreitet. Zudem wird dadurch der Kostenaufwand minimiert.
+	   
 	   
 	\section{Struktur}
-	
-	\begin{enumerate}
-	    \item Test
-	    \item Bla
-        \begin{enumerate}
-        	    \item Test
-        	    \item Bla
-        	\end{enumerate}
-	\end{enumerate}
+    		Die Struktur der Arbeit ist wie folgt vorgesehen:
     
+    		\paragraph{Was ist Open Source Software?}
+    		Zunächst wird ein kurzer Überblick über Open Source Software gegeben. Dazu gehört die Definition und die heutige Verwendung.
+		
+		\paragraph{Nutzung von Open Source Software von Endanwendern}
```

Comment:
This paragraph should include the results of the survey. Note it in the text one way or another but it needs to be in there.